### ISSUE TITLE
'qiime metadata tabulate' command is wrong only 'taxonomy.qza' to 'taxonomy.qzv'

### ISSUE THREAD

**Issue by hyoj-seo** [qiime2/qiime2#618]
Hello, I'm currently using qiime2. Thank you for the good tools.

Currently, I am experiencing something strange about the `qiime metadata tabulate` command.

The qza file created through the denoising process is well converted to qzv and leaves the following log.
```
qiime dada2 denoise-paired \
--p-n-threads 40 \
--i-demultiplexed-seqs 1.Import_data/Manifest.qza \
--p-trunc-len-f 277 \
--p-trunc-len-r 222 \
--p-trim-left-f 13 \
--p-trim-left-r 5 \
--o-representative-sequences 2.Denoising/rep_seqs.qza \
--o-table 2.Denoising/table.qza \
--o-denoising-stats 2.Denoising/stats_dada2.qza \
--verbose
```
```
qiime metadata tabulate \
--m-input-file 2.Denoising/stats_dada2.qza \
--o-visualization 2.Denoising/stats_dada2.qzv
```
`Saved Visualization to: stats_dada2.qzv`

However, the qza file created after 'quiime feature-classifier classify-sklearn' leaves the following logs, and the qzv file is not saved.
```
qiime feature-classifier classify-sklearn \
--i-classifier 0.DB/silva-138-99-nb-classifier.qza \
--i-reads 2.Denoising/rep_seqs.qza \
--p-n-jobs 20 \
--o-classification 3.Classification/taxonomy.qza
```
```
qiime metadata tabulate \ 
--m-input-file 3.Classification/taxonomy.qza \
--o-visualization 3.Classification/taxonomy.qzv
```
`Saved FeatureData[Taxonomy] to: 3.Classification/taxonomy.qza`

I tried `--output-dir` instead of `--o-visualization` of the 'qiime metadata tabulate', but it was still not saved, and the relevant files were not saved using the qiime tools export.

**The problem only occurs in qza generated after classify-sklearn!**

The qiime2 currently in use is 2021.11 version, and I tried it as 2021.8 version, but there was no difference.

Do you know how to solve it?
Is it over the command or is it a program problem?
After the update, taxonomy.qza no longer supports qzv?

Thank you for reading the long text.
Thank you for your answer in advance.

**Comment by ebolyen** [comment]
Hi @hyoj-seo,

I think this question should probably go to the [QIIME 2 Forum](https://forum.qiime2.org) as I'm not able to reproduce, so there's probably something else going on.

I set up a similar situation and saw this:
```
§ qiime metadata tabulate \
> --m-input-file 3.Classification/taxonomy.qza \
> --o-visualization 3.Classification/taxonomy.qzv

Saved Visualization to: 3.Classification/taxonomy.qzv
```

---

Is there any chance you hit the &lt;tab> key while filling in `--o-visualization`? I did that almost instinctively while running the above and it auto-completes with `3.Classification/taxonomy.qza` which would match what you see as output from the command.

**Comment by hyoj-seo** [comment]
Thank you for your quick response, @ebolyen!

When I organized the script, I used `\`, but when I actually ran, I connected it in one line.
As follows.
`qiime metadata tabulate  --m-input-file 3.Classification/taxonomy.qza  --o-visualization 3.Classification/taxonomy.qzv`

**I can't believe it, but when I ran with the command you wrote me, I created a great qzv file!** (Running directly on linux. Previously, it was written on python and printed out.)
But if you make your code in a single line, it doesn't generate qzv.
It's a strange problem, but we created qzv, so it's successful.

Once again, thank you for your quick help.
hyoj-seo.
